projects/plugins/social/src/js/components/header/index.js

```diff
@@ -63,6 +66,12 @@ const Header = () => {
 								loading: null === sharesCount,
 								value: formatter.format( sharesCount ),
 							},
+							{
+								icon: <Icon icon={ postList } />,
+								label: __( 'Posted this month', 'jetpack-social' ),
```

Comment:
I know this is what's in the designs but I wonder if 'Posted this month' should be 'Posts shared this month'. It doesn't seem clear to me that it's about posts and specifically posts that you've publicized. This would mean we'd want to use `_n` to cope with when there's only one post (something that we might also want to do with the total shares stat too).

Reply:
Yeah, I like that. We can do that in a follow-up PR. As an aside, I don't think we'd need to use `_n` to differentiate, because the label is separated from the number. To me, `Posts shared this month: 1` is preferred, but `1 posts shared this month` would be wrong.

Reply:
Yeah that's true. I think because the number comes first I was reading it the second way.

Reply:
It doesn't come first! What am I on about?